### ISSUE TITLE
Show close buttons effective only after restart

### ISSUE THREAD

**Issue by Nonoreve** [geany/geany#3167]
The checkbox "Show close buttons" in "Notebook tabs" in Preferences only take effect after a restart of geany.
tested under Geany 1.38 Using GTK+ v3.24.33 and GLib v2.70.4 and also with origin/master version
Os Manjaro Linux

**Comment by elextr** [comment]
No, it works immediately, all new windows will follow the preference, but it does not modify existing windows when the preference changes.

A well written PR to add that would likely be accepted.
